app/src/main/java/unipd/se18/ocrcamera/MainActivity.java

```diff
@@ -0,0 +1,123 @@
+package com.example.group4.cameraapp;
+
+import android.Manifest;
+import android.content.Intent;
+import android.content.pm.PackageManager;
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.net.Uri;
+import android.os.Environment;
+import android.provider.MediaStore;
+import android.support.v4.app.ActivityCompat;
+import android.support.v4.content.ContextCompat;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.ImageView;
+import android.widget.TextView;
+
```

Comment:
from the StyleDoc: (If there are both static and non-static imports, a single blank line separates the two blocks), there are no other blank lines between import statements.